Makefile

```diff
@@ -184,7 +184,6 @@ endif
 # Prepare optional Boards build.
 BOARDS_PACKAGES=$(shell $(GO) list $(BUILD_BOARDS_DIR)/server/...)
 ifeq ($(BUILD_BOARDS),true)
-    ALL_PACKAGES += $(BOARDS_PACKAGES)
```

Comment:
If we include board packages in Allpackages , test will try to execute those tests which needs `-tag 'json1'`. How ever whenever we add that flag mattermost-server builds and tests are failing with unsupported tag error.

Reply:
Let's add a comment to explain why we aren't doing this so that people are aware (possibly along with a ticket reference)

Reply:
Comment is added with PR and ticket reference.